### DISCUSSION TITLE
pancake bot

### DISCUSSION THREAD

**Discussion by Fulopop1** [storycraft/node-kakao#757] [Q&A]
팬케이크 봇 실행하는법좀 알려주시면 감사하겠습니다.

**Comment by storycraft** [comment]
저장소를 클론한뒤 npm install로 필요라이브러리 설치후 npm start로 시작하시면 됩니다. 지속적인 기능 추가로 readme.md에 쓰여있지 않은 모듈이 몇개 있습니다. (이모티콘 추출기, tts, 챗봇 등) 필요없는 모듈이나 필요한 환경변수는 저장소 최상단에 있는 index.ts 파일을 참고하거나 수정해 시작전 조정하세요.
